app/src/main/java/com/amaze/filemanager/ui/activities/superclasses/PreferenceActivity.java

```diff
@@ -57,9 +57,9 @@
 
   @Override
   public void onCreate(final Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
-
     sharedPrefs = PreferenceManager.getDefaultSharedPreferences(this);
+
+    super.onCreate(savedInstanceState);
```

Comment:
Why?!

Reply:
Because fragments are created before the `super` call returns, so we must initialize `sharedPrefs` before the `super` call otherwise it cannot be used by fragments